### DISCUSSION TITLE
How add custom property to ICurrentUser

### DISCUSSION THREAD

**Discussion by genoher** [abpframework/abp#9874] [Q&A] [answered]
I try to customize my AppUser with some extra properties.

I like to add these properties to injected ICurrentUser of AppService.

How i can do that? i don't see any article explain that... 

I follow the article [How to add custom property to the user entry](https://community.abp.io/articles/how-to-add-custom-property-to-the-user-entity-6ggxiddr) 

Thanks a lot and sorry for my english...

**Comment by maliming** [comment, marked as answer]
See https://github.com/abpframework/abp/issues/2930#issuecomment-591756387
https://github.com/abpframework/abp/pull/8153/files#diff-d73ab7794fbad910dc395d2188bdf3692f9a312a4efa69cbdb5cf7be41129ee3R45

**Reply by genoher** [reply]
I check the links and I try to implement. Thanks @maliming ;)
